### PR TITLE
Check for object exist

### PR DESCRIPTION
**Pull request by JERILJOSE96**

### CODE REVIEW

**Review by espresse — APPROVED** [review]
LGTM :+1:

File: src/content-fusion/content-fusion.ts [hunk 1]
```diff
@@ -19,6 +22,11 @@ export class ContentFusion {
         const localization = res.localization[locale];
         res.localization = {};
         res.localization[locale] = localization;
+      } else {
+        throw <HttpClientProxyError>{
+          message: 'Request failed with status code 404',
```
Comment (espresse) [inline src/content-fusion/content-fusion.ts]: @JERILJOSE96 @iapain I'm not sure this is a correct behaviour of the client
Anyway, if it is there should be some info in the thrown object to specify why there's 404.

Comment (iapain) [reply src/content-fusion/content-fusion.ts]: @espresse I was expecting `axios` will throw HTTPException by itself.

File: test/erudio-client.spec.ts [hunk 2]
```diff
@@ -27,27 +27,58 @@ describe('Throws exemption for', () => {
...
+    mock
+      .onGet(`${structureService}${namespace}/nodes/${structureID}`)
+      .replyOnce(200, singleNode);
+    mock
+      .onGet(`${structureService}${namespace}/children/nodes/${structureID}`)
+      .replyOnce(200, nodeList);
+
+    mock.onGet(`${contentFusionService}${childNodeID}`).reply(200, content);
+    const allStructureData = ec.getStructureNode(namespace, structureID, 'nb');
+    await expect(allStructureData).rejects.toMatchObject({
```
Comment (iapain) [inline test/erudio-client.spec.ts]: @JERILJOSE96 Why this test returns exception, in your mock it returns 200

**Review by iapain — CHANGES_REQUESTED** [review]

**Review by iapain — APPROVED** [review]

**Pull request merged**